septentrion/configuration.py

```diff
@@ -188,6 +188,8 @@ def load_configuration_files(value: Optional[TextIO]) -> Dict[str, Any]:
         logger.info(f"Reading configuration from {file}")
 
         file_contents = value.read()
+        if isinstance(file_contents, bytes):
+            file_contents = file_contents.decode()
```

Comment:
I think I'd rather not change this and instead fix this:
https://github.com/peopledoc/septentrion/blob/master/septentrion/cli.py#L80

Reply:
I thought about this at night too :)

Reply:
Done and ready for review